content/20211219-dev-meetups.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Building a software developer community"
+slug: "building_a_software_developer-community"
+cover: "016_dev_meetups.png"
```

Comment:
could you change the cover tag?

Likewise the "slug" needs to be `-` dashes 

---
title: "Javascript Features To Use in 2021"
slug: "javascript-features"
cover: "code_chefs_podcast_art.png"
date: 2021-12-19
audioPath: 015_javascript_features.mp3
episodeNumber: 15
fileSize: 124
showLength: 54:16
category: "tech"
# shortDescription is only for codechefs.dev homepage
shortDescription: "JavaScript has taken over the world over the past decade. Come learn about its history and the latest features."
tags:
  - programming
  - javascript
---

Reply:
use `cover: "code_chefs_podcast_art.png"`

And slug make it hypenated